docs/docs/usage/index.mdx

```diff
@@ -39,6 +39,45 @@ clouder ssh-key ls
 clouder vm ls
 ```
 
+## Context
+
+```bash
+clouder ctx init
+clouder ctx
+clouder ctx set ovh edda3...
+```
+
+## Kubernetes
+
+```bash
+clouder k8s create $DATALAYER_CLUSTER_NAME
+clouder k8s kubeconfig $DATALAYER_CLUSTER_NAME
+clouder k8s
+clouder k8s use $DATALAYER_CLUSTER_NAME
+clouder k8s
+clouder info
+```
+
+```bash
+clouder k8s create-nodepool $DATALAYER_CLUSTER_NAME router --flavor b2-15 --min 0 --desired 1 --max 10 --role router --roles router --xpu cpu
```

Comment:
is `--roles <name>` ok or is the `=` sign mandatory?

Reply:
trying now, indeed, it is not clear...

Reply:
I confirm it is working without the `=` , so the PR is good to me.

Reply:
Thanks for quickly testing